test/helper/sending.bash

```diff
@@ -82,20 +132,35 @@ function _send_email() {
 # chosen. Sending more than one mail at any given point in time with this function
 # is UNDEFINED BEHAVIOR!
 function _send_email_and_get_id() {
-  [[ -v CONTAINER_NAME ]] || return 1
-
-  _wait_for_empty_mail_queue_in_container
-  _send_email "${@}"
-  _wait_for_empty_mail_queue_in_container
+  # Export the variable denoted by ${1} so everyone has access
+  export "${1:?Mail ID must be set for _send_email_and_get_id}"
+  # Get a "reference" to the content of the variable denoted by ${1} so we can manipulate the content
+  local -n ID_ENV_VAR_REF=${1:?}
+  # Prepare the message ID header here because will will shift away ${1} later
+  local MID="<${1}@dms-tests>"
+  # Get rid of ${1} so only the arguments for swaks remain
+  shift 1
 
-  local MAIL_ID
+  local QUEUE_ID MESSAGE_ID
   # The unique ID Postfix (and other services) use may be different in length
   # on different systems (e.g. amd64 (11) vs aarch64 (10)). Hence, we use a
   # range to safely capture it.
-  MAIL_ID=$(_exec_in_container tac /var/log/mail.log              \
-    | grep -E -m 1 'postfix/smtpd.*: [A-Z0-9]+: client=localhost' \
-    | grep -E -o '[A-Z0-9]{9,12}' || true)
+  local QUEUE_ID_REGEX='[A-Z0-9]{9,12}'
```

Comment:
Update comment? Not likely due to aarch64.

Reply:
I saw a different length on my aarch64 (arm64) system...

Reply:
> I saw a different length on my aarch64 (arm64) system...

Did you miss the context comment I provided earlier? The length is likely due to inode number encoded being larger.

That can just be due to the inode number issued, or filesystem which uses a larger inode number (_64-bit vs 128-bit for example_).

Just because you saw it on ARM64 environment doesn't mean it was actually due to the platform difference.

Reply:
Ah, I understand. I will change it then 👍🏼